yarn.lock

```diff
@@ -2513,10 +2548,18 @@
     "@opentripplanner/core-utils" "^7.0.1"
     prop-types "^15.7.2"
 
-"@opentripplanner/itinerary-body@^4.0.4":
-  version "4.0.5"
-  resolved "https://registry.yarnpkg.com/@opentripplanner/itinerary-body/-/itinerary-body-4.0.5.tgz#6970bcf6733f0972324cdde57e9dbd7660411b9e"
-  integrity sha512-rQKCZTDKXb+nj+bj/WLnX7o2n3sMRWPtHtZ5Vp5ndi5jjf1xxbMXXaOEjDBrquQkLzWK/Vg0Fln37cnDlTvvaw==
+"@opentripplanner/icons@^1.2.6":
+  version "1.2.6"
+  resolved "https://registry.yarnpkg.com/@opentripplanner/icons/-/icons-1.2.6.tgz#8222c089a1c498c93fd5d7cf2affb72eb6ffdcdc"
+  integrity sha512-8ZRbottKNpV4wV6WJeVAuMRyOAc8fYtGjFDLtYpDVJLodKoCvs/GkMxP6upNgLaMvsppAHdXtCn58Djk8Rkkeg==
+  dependencies:
+    "@opentripplanner/core-utils" "^7.0.3"
+    prop-types "^15.7.2"
+
+"@opentripplanner/itinerary-body@^4.1.1":
```

Comment:
Can you `yarn-deduplicate` this reference (and others)?

Reply:
Good call! Run in bf708664b15bebcbcff188c849ad5ee3e75b4795